webrtc.html

```diff
@@ -12928,6 +12928,51 @@ <h2>Setting SDP from remote endpoints</h2>
       SDP.</p>
     </section>
   </section>
+  <section class="informative">
+    <h2>Accessibility Considerations</h2>
+    <p>The WebRTC 1.0 specification exposes an API to control protocols
+      (defined within the IETF) necessary to establish real-time
+      audio, video and data exchange.</p>
+
+    <p>Real-time Text, defined in [[RFC4103]] is used by people who are hearing
```

Comment:
We can't be this positive about it. RFC 4103 is being obsoleted (as later text says), so we shouldn't write of it as if it was "up and coming".

Reply:
How about this alternative paragraph?

"The Telecommunications Device for the Deaf (TDD/TTY) enables
      individuals who are hearing or speech impaired (among others)
      to communicate over telephone lines. Real-time Text, defined
      in [[RFC4103]], utilizes T.140 encapsulated in RTP to enable
      the transition from TDD/TTY devices to IP-based communications,
      including emergency communication with
      <a href="https://www.ecfr.gov/cgi-bin/text-idx?SID=ada7cebd65433a9e05c35426a2bc76b8&amp;mc=true&amp;node=pt47.3.67&amp;rgn=div5">Public Safety Access Points (PSAP)</a>"